jgiven-junit5/src/test/java/com/tngtech/jgiven/junit5/test/JUnit5ScenarioTest.java

```diff
@@ -12,4 +12,5 @@ public void JGiven_works_with_JUnit5_DISABLED() {
         then().some_outcome();
     }
 
+
```

Comment:
Two lines? :D

Reply:
damn...
not gonna fix that though. really not worth it